packages/angular-material/src/layouts/group-layout.renderer.ts

```diff
@@ -34,19 +34,34 @@ import { JsonFormsAngularService } from '@jsonforms/angular';
 @Component({
   selector: 'GroupLayoutRenderer',
   template: `
-    <mat-card appearance="outlined" fxLayout="column" [fxHide]="hidden">
+    <mat-card
+      [ngStyle]="{ display: hidden ? 'none' : '' }"
+      appearance="outlined"
+      class="group-layout"
+    >
       <mat-card-title class="mat-headline-6">{{ label }}</mat-card-title>
       <div
         *ngFor="
           let props of uischema | layoutChildrenRenderProps : schema : path;
           trackBy: trackElement
         "
-        fxFlex
       >
         <jsonforms-outlet [renderProps]="props"></jsonforms-outlet>
       </div>
     </mat-card>
   `,
+  styles: [
+    `
+      .group-layout {
```

Comment:
I think the .group-layout class misses a `padding: 16px;`. Before there was some distance to the borders of the group. This is gone in this PR. The padding in before this was 16px.

Reply:
It's an artifact from the previous PR but indeed, `.mat-card` had a padding of 16px before. I'll fix it.